### DISCUSSION TITLE
Offset

### DISCUSSION THREAD

**Discussion by jgomez720** [KittyCAD/modeling-app#512] [New Feature Requests]
### Description
A tool that enables the user to create a new sketch entity by offsetting from an existing one.

### Functionality
1. The user selects the _Offset_ button.
2. The user selects the sketch entities they want to offset from.
3. The user sets the dimension of the offset and the direction.
4. Confirm.

### Supporting Docs/Images
https://github.com/KittyCAD/modeling-app/assets/114548659/ec44194f-ab40-493a-b121-5988e67ea6dd

**Comment by yeroca** [comment]
I second this.  Another source for the offset feature is what OpenSCAD has: https://en.wikibooks.org/wiki/OpenSCAD_User_Manual/Transformations#offset  which allows what is demonstrated in the video, but also an ability to create radiuses around the corners.
